src/language/texts/nb.ts

```diff
@@ -293,6 +293,11 @@ export function nb(): FixedLanguageList {
         payer: 'Betaler',
         name: 'Navn',
         phone: 'Telefon',
+        company_name: 'Firmanavn',
+        org_number: 'Organisasjonsnummer',
+        contact_person: 'Kontaktperson',
+        contact_phone: 'Kontakt telefon',
+        contact_email: 'Kontakt e-post',
```

Comment:
Why is there a difference in spacing between `nb` and `nn`?

Reply:
I would think that no spacing is correct?

Reply:
Yep, that was a mistake. no spacing is correct 👍